front/app/containers/Admin/projects/project/analysis/TopBar/Filters/index.tsx

```diff
@@ -21,8 +31,13 @@ const Filters = () => {
       left="0"
       bg={colors.white}
       w="100%"
-      p="24px"
+      px="24px"
+      pb="24px"
+      boxShadow={defaultStyles.boxShadow}
     >
+      <Box alignSelf="flex-end" position="absolute" right="16px" top="16px">
```

Comment:
Does alignSelf still do anything when postition is absolute?

Reply:
Nope, I don't think it works. Probably left over from when I tried to align it differently. :)